### DISCUSSION TITLE
Reverifying GitHub Education Pack

### DISCUSSION THREAD

**Discussion by MouzinGulzar** [community/community#40686] [GitHub Education]
I got approved last year for GitHub Education Pack and recently that expired. Now I want to reverify it but there it shows me you are already verified no need to verify again. But I ain't able to access the benifits. What can be done now?

**Comment by samus-aran** [comment]
Hi @MouzinGulzar 👋 

Can you provide more information on how you are unable to redeem your GitHub Student benefits? Are you receiving any errors?
